manager/manager.go

```diff
@@ -247,27 +247,38 @@ type Manager struct {
 }
 
 // DumpMaps - Return a string containing human readable info about eBPF maps
-func (m *Manager) DumpMaps() (string, error) {
+// is not maps list given all maps with an DumpHandler set are dumped
```

Comment:
Maybe reword to "Dumps the set of maps provided, otherwise dumping all maps with a DumpHandler set."

Reply:
done